### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by ZhangLupeng123** [kajjohns/SliDeFS#2]
Hi, Johnson, I am Lupeng Zhang, from the Institute of Earthquake Forecasting, China. In your code, does each fault segment in the input data have a specific seismic recurrence time, correct? However, in your code, I noticed a line that sets the recurrence time. After running this line of code (%recurrence time t_eq_frac = 318;), the result is that all fault segments have the same seismic recurrence time. I would like to ask what the reason for this is, or if my understanding is incorrect, please clarify.

**Comment by ncperdomo** [comment]
Hi Lupeng

The recurrence time for each fault segment can be specified in the FaultInfo structure:

FaultInfo{k}.RecurrenceTime should contain the earthquake recurrence time in years, you can set it to 'nan' if you don't want to compute earthquake cycles for one or more fault segments (i.e., use the elastic halfspace solution). To use the default value, you can also specify 'inf' as we did in the code.

**Comment by ZhangLupeng123** [comment]
I am indeed trying to calculate the fault slip rate using a viscoelastic relaxation model that considers the seismic cycle. However, in your code, you can only set the Maxwell viscoelastic body, but not the Burger viscoelastic body, correct?

**Comment by ncperdomo** [comment]
Yes, we only consider a Maxwell viscoelastic rheology. Also, I'd like to point out that the code estimates slip deficit rates, not slip rates on the 3D fault patches.

**Comment by ZhangLupeng123** [comment]
Thanks!
